cfme/configure/access_control.py

```diff
@@ -837,8 +1041,15 @@ def create(self, cancel=False):
         """ Create role method
             Args:
                 cancel: True - if you want to cancel role creation,
-                        by defaul, role will be created
+                        by default, role will be created
+        Throws:
+            RBACOperationBlocked: If operation is blocked due to current user
+                not having appropriate permissions OR update is not allowed
+                for currently selected role
         """
+        flash_blocked_msg = VersionPick({
+            '5.7': ("Name has already been taken")}).pick(self.appliance.version)
```

Comment:
It's removed it. It was a holdover from a 5.6 test string. I left the version pick in as a quick way to easily insert a string version update.